php/libraries/NDB_Notifier.class.inc

```diff
@@ -85,20 +85,21 @@ class NDB_Notifier extends NDB_Notifier_Abstract
     private function _sendEmails(): void
     {
         foreach ($this->notified['email_text'] as $name=>$uid) {
+            $notifierID = $this->notifier->getData('ID');
+            $fromEmail  = $this->notifier->getData('Email');
             $this->tpl_data['notified_user'] = $name;
-            $this->tpl_data['notifier_user']
-                = $this->notifier['name'].' <'.$this->notifier['email'].'>';
-            $email =$this->getEmail($uid);
+            $this->tpl_data['notifier_user'] = "$notifierID <$fromEmail>";
+            $toEmail = $this->getEmail($uid);
 
-            if ($this->notifier['ID'] === $uid) {
+            if ($notifierID === $uid) {
                 // TODO send confirmation email instead
             } else {
                 Email::send(
-                    $email,
+                    $toEmail,
```

Comment:
Everything looks great! My only suggestion is maybe replacing $toEmail with $replyEmail but that's just my preference and $toEmail works.

Reply:
or maybe even $replyEmailAddress

Reply:
It's actually the other way around. The toEmail is the recipient and the fromEmail is the sender. I'll change the variable names though. You're right, they could be better.